extensions/ci/main.ts

```diff
@@ -0,0 +1,44 @@
+import { PhylumApi } from "phylum";
+
+// Ensure required arguments are present.
+if (Deno.args.length < 4 || Deno.args.length > 5) {
+  console.error(
+    "Usage: phylum ci <PROJECT> [--group <GROUP>] <LABEL> <BASE> <LOCKFILE>",
+  );
+  Deno.exit(1);
+}
+
+// Find optional groups argument.
+let group = undefined;
+let groupArgsIndex = Deno.args.indexOf("--group");
```

Comment:
```suggestion
const groupArgsIndex = Deno.args.indexOf("--group");
```
To satisfy `deno lint`.

Reply:
Already done.